core/Model/CalendarModel.vala

```diff
@@ -294,6 +290,38 @@ public class Maya.Model.CalendarModel : Object {
 
     //--- Helper Methods ---//
 
+    /** Set the week_starts_on property: the first day of the week.
+     *
+     * Locale handling is based on information from
+     * https://sourceware.org/glibc/wiki/Locales
+     */
+    private GLib.DateWeekday get_week_start () {
+        // Set the "baseline" for start of week: Sunday or Monday?
+        uint week_day1 = (uint) Posix.NLTime.WEEK_1STDAY.to_string ();
+        debug (@"WEEK_1STDAY: $(week_day1)");
+        var week_1stday = 0; // Default to 0 if unrecognized data
+        if (week_day1 == 19971130) { // Sunday
+            week_1stday = 0;
+        } else if (week_day1 == 19971201) { // Monday
+            week_1stday = 1;
+        } else {
+            warning (@"Unknown value of _NL_TIME_WEEK_1STDAY: @(week_day1)");
```

Comment:
The linter isn't happy about this. Could use printf syntax instead:
```suggestion
            warning ("Unknown value of _NL_TIME_WEEK_1STDAY: %s", week_day1);
```

Reply:
Interesting. Thanks! Is there something different about the vala-lint we're running on CI vs the one on apt? I have it on my computer, but when I run `io.elementary.vala-lint -d core` (or `core/Model` or just about any other directory) it doesn't give me any errors. I have the latest version from apt.